modules/ob-creating-a-shipwright-build-resource-console.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assembly:
+//
+// builds/installing-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-shipwright-build-resource-console_{context}"]
+= Creating a ShipwrightBuild resource by using console
+
+After installing the {builds-title} Operator, you must create a `ShipwrightBuild` resource to enable the functioning of the build controller.
+
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to the *Operators -> Installed Operators* page.
+. Click the {builds-title} Operator link present in the list. The *Operator details* page opens.
+. Select the *Shipwright Build* tab and click *Create ShipwrightBuild*.
+. Select either the *Form view* or *YAML view* to configure a new `ShipwrightBuild` resource in the following manner:
+** On selecting the *Form view* or *YAML view*, you get to see the default values configured for the `name` and `targetNamespace` fields. If you do not want to edit those fields, click *Create* to configure the `ShipwrightBuild` resource with default values.
++
+You can view the created resource in the *Shipwright Build* tab.
```

Comment:
Should we add a `Verification` Section here as well? which states.
~~~
A controller pod in the mentioned targetNamespace must have been created.
~~~